model_atlas_attribute_def.go

```diff
@@ -1,29 +1,723 @@
+// Copyright 2021 Confluent Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
 /*
- * Confluent Schema Registry
- *
- * No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
- *
- * API version: v1
- * Generated by: OpenAPI Generator (https://openapi-generator.tech)
- */
+Confluent Schema Registry
+
+No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
+
+API version: v1
+*/
+
+// Code generated by OpenAPI Generator (https://openapi-generator.tech); DO NOT EDIT.
 
 package schemaregistry
+
+import (
+	"encoding/json"
+)
+
+import (
+	"reflect"
+)
+
 // AtlasAttributeDef struct for AtlasAttributeDef
 type AtlasAttributeDef struct {
-	Name string `json:"name,omitempty"`
-	TypeName string `json:"typeName,omitempty"`
-	IsOptional bool `json:"isOptional,omitempty"`
-	Cardinality string `json:"cardinality,omitempty"`
-	ValuesMinCount int32 `json:"valuesMinCount,omitempty"`
-	ValuesMaxCount int32 `json:"valuesMaxCount,omitempty"`
-	IsUnique bool `json:"isUnique,omitempty"`
-	IsIndexable bool `json:"isIndexable,omitempty"`
-	IncludeInNotification bool `json:"includeInNotification,omitempty"`
-	DefaultValue string `json:"defaultValue,omitempty"`
-	Description string `json:"description,omitempty"`
-	SearchWeight int32 `json:"searchWeight,omitempty"`
-	IndexType string `json:"indexType,omitempty"`
-	Constraints []AtlasConstraintDef `json:"constraints,omitempty"`
-	Options map[string]string `json:"options,omitempty"`
-	DisplayName string `json:"displayName,omitempty"`
+	Name *string `json:"name,omitempty"`
```

Comment:
@xiaoyali97 fields in the struct have been changed from value to a pointer, and if someone updates their sdk this could break their code. Is that ok?

Reply:
cc @rayokota